docs/0.5.0-alpha/getting_started/usage.md

```diff
@@ -93,15 +93,34 @@ Furthermore, Amber adds a _shebang_ at the top of the compiled script. This enab
 $ ./output.sh
 ```
 
-#### Postprocessors
+## Syntax Highlighting
```

Comment:
I moved this section higher since it's more interesting than how to minify and configure amber.